src/header.c

```diff
@@ -2781,6 +2781,13 @@ dxf_header_read
                  * variables stored as an int, but that can have up
                  * to sixteen hexadecimal digits (64 bits) */
 
+                /* Good news: the DXF reference provided by Autodesk
+                 * is only accurate to AutoCAD 2012. There are
+                 * header variables in a DXF I just created that are
+                 * nowhere to be found in the DXF reference. I can't
+                 * find any information on DXF file format
+                 * specifications more recent than 2012. */
```

Comment:
I do happen to have found a 2014 version of the DXF reference guide ... can't find the URL ... must still be somewhere out there for grabs ;-)